extensions/src/services/http-svc.js

```diff
@@ -0,0 +1,32 @@
+
+import axios from 'axios';
+
+/**
+ * This will be the wrapper of actual axios library so we can add
+ * authorization headers for all the extensions.
+ */
+
+let axiosInstance = null;
+
+function setNexusAuth(token) {
```

Comment:
This method isn't really connected to nexus as tokens will be set for any request, so no need to mention nexus here. Also setAuth sounds like you want to set a method of authorisation or so, to be more specific I would add `Token` word somewhere.